doc/source/download.rst

```diff
@@ -10,281 +10,38 @@ Download
        :depth: 3
        :backlinks: none
 
+The GDAL project distributes GDAL as source code and :ref:`Containers` only. :ref:`Binaries` produced by others are available for a variety of platforms and package managers.
+
+Source Code
+-----------
+
 Current Release
-------------------------------------------------------------------------------
+...............
 
 * **2024-05-10** `gdal-3.9.0.tar.gz`_ `3.9.0 Release Notes`_ (`3.9.0 md5`_)
 
 .. _`3.9.0 Release Notes`: https://github.com/OSGeo/gdal/blob/v3.9.0/NEWS.md
 .. _`gdal-3.9.0.tar.gz`: https://github.com/OSGeo/gdal/releases/download/v3.9.0/gdal-3.9.0.tar.gz
 .. _`3.9.0 md5`: https://github.com/OSGeo/gdal/releases/download/v3.9.0/gdal-3.9.0.tar.gz.md5
 
-Past Releases
-------------------------------------------------------------------------------
+Additional Supported Releases
```

Comment:
This should also be moved to past releases. We only support the 3.9 series at that point

Reply:
done